reference/roadmap.md

```diff
@@ -14,3 +14,4 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 |---------|-------------|---------------|
 | 1.21.1 | March 13 2024 | * Android security updates for March 2024<br/>* Bug fixes |
 | 1.21.2 | April 17 2024 | * Android security updates for April 2024<br/>* Bug fixes |
+| 1.22.0 | May 15 2024 | * Support for custom Android system images<br/> * Support for 64 bit only ARM/x86 systems<br/>* Stability improvements for AAOS images and VHAL |
```

Comment:
Can we add `VirGL enabled by default for NVIDIA GPUs` and `Update to NVIDIA 550 driver series`?

Reply:
Done